src/mapping/mappers/Partners.js

```diff
@@ -0,0 +1,33 @@
+const { AggregateMapper } = require('../mapper');
+const { applyProfile, mcodeUtils09 } = require('../../utils');
+
+const resourceMapping = {
+    filter: () => true,
+    default: (resource, _context) => applyProfile(resource, mcodeUtils09.defaultProfile(resource.resourceType)),
+    mappers: [
+        {
+            filter: 'AllergyIntolerance',
+            exec: (resource, context) => {
+                if (!resource.substance.coding) {
```

Comment:
Minor comment: We may also want to check here that there is a text present. On the one hand, there almost certainly is one since there has to be a CodeableConcept present for the allergyInt. On the other hand, we might as well make sure that we're not throwing `undefined` in our mapped patient record. Let's check that the text is there, and if it isn't we'll just use an empty string here.

Reply:
Resolved